### PR TITLE
Add_Validator_In_User_Service_And_Tests For_It

### PR DESCRIPTION
**Pull request by DanilFadeeev**

### CODE REVIEW

**Review by Cerberous007 — COMMENTED** [review]
Please rename pull request. If you have any questions, check out our code convention.

**Review by Cerberous007 — COMMENTED** [review]
What happens to this branch? You should make merge develop into your branch

**Comment by DanilFadeeev** [comment]
> What happens to this branch? You should make merge develop into your branch

i did

**Comment by dTarasova** [comment]
Name is still doesn't match code convention :)

Sorry , I don't get it . A lot of changed files and I think some work repeats existing files

File: Common/DTO/BrokerRequests/InternalDeleteUserRequest.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+
+namespace DTO.BrokerRequests
+{
+    public class InternalDeleteUserRequest
+    {
+        public Guid UserIdCredential { get; set; }
```
Comment (dTarasova) [inline Common/DTO/BrokerRequests/InternalDeleteUserRequest.cs:8]: This aready has to be UserId :)

File: Services/UserService/Controllers/UsersController.cs [hunk 2]
```diff
@@ -19,6 +21,16 @@ public class UsersController : ControllerBase
         [HttpPost]
         public async Task<bool> CreateUser([FromServices] ICreateUserCommand command, [FromBody] CreateUserRequest request)
         {
+            CreateUserRequesValidator validator = new CreateUserRequesValidator();
```
Comment (dTarasova) [inline Services/UserService/Controllers/UsersController.cs:24]: What? Nononono
We already have data validation in createUserCommand
Why do you have to write all logic here?!

File: SingUpModelValidationTests/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
+using NUnit.Framework;
+
+namespace SingUpModelValidationTests
+{
+    public class Tests
```
Comment (dTarasova) [inline SingUpModelValidationTests/UnitTest1.cs:5]: Are you sure that we need these tests?

File: Tests/AuthServiceTests/Validators/CreateUserRequestValidatorTests.cs [hunk 4]
```diff
@@ -0,0 +1,130 @@
+﻿using AuthenticationService.Utils;
+using DTO.RestRequests;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace AuthenticationServiceTests.Validators
```
Comment (dTarasova) [inline Tests/AuthServiceTests/Validators/CreateUserRequestValidatorTests.cs:8]: In Authentification. CreateUserRequestTests. Are you seriously?

Comment (dTarasova) [reply Tests/AuthServiceTests/Validators/CreateUserRequestValidatorTests.cs:8]: Also, we already have sth like that in  user service" createuserrequestvalidator

**Review by dTarasova — CHANGES_REQUESTED** [review]

File: Common/DTO/BrokerRequests/InternalDeleteUserRequest.cs (hunk 1 above)
Comment (nmamro) [reply Common/DTO/BrokerRequests/InternalDeleteUserRequest.cs:8]: > This aready has to be UserId :)

Technically, this hasn't been pushed to develop yet, has it?

File: Common/DTO/ErrorResponse.cs [hunk 5]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
```
Comment (nmamro) [inline Common/DTO/ErrorResponse.cs:1]: I don't understand why these are in **Files changed** list, any ideas?

File: PostmanTests/UserService.postman_collection.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
```
Comment (nmamro) [inline PostmanTests/UserService.postman_collection.json:1]: I don't understand why these are in **Files changed** list, any ideas?

**Review by nmamro — CHANGES_REQUESTED** [review]
Please, check why **Files changed** list is so strange

**Pull request closed without merging**